contrib/analysis/generate-pbench-timeseries-graphs

```diff
@@ -59,258 +59,264 @@ parser.add_argument('files', metavar='file', type=str, nargs="*", help="files")
 args = parser.parse_args()
 
 pattern_map = {
-    re.compile('disk_IOPS.csv$') : {
+    re.compile('disk_IOPS\\.csv$') : {
```

Comment:
Using "r" strings might help here, see https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals

Reply:
We also have code like this already in the pbench server: https://github.com/distributed-system-analysis/pbench/blob/master/lib/pbench/server/indexer.py#L1847

Reply:
That code looks close to what I want.  I'd also like to have a title, graph type (or is that implied in any of the other graphs?), and optional axis labels.  But it would certainly make sense to reuse it.

Reply:
And I fixed up the regexps.